backend/controllers/UserController.ts

```diff
@@ -27,7 +27,37 @@ const UserController = {
     } catch (err: any) {
       return res.status(500).json({ error: err.message });
     }
-  }
+  },
+  async changeUserData(req: Request, res: Response) {
+    try {
+      let { id, ...newUserData } = req.body;
+      const imageUrl:any = req.file
+      if(imageUrl){
+        newUserData= {
+          ... newUserData,
+          imageUrl:imageUrl.location
```

Comment:
we should configure your prettier and eslint because - there are so many examples without gaps)